delete_account.php

```diff
@@ -58,10 +58,12 @@
 		
 		<!-- The form to delete an account -->
 		<div class="account_forms">
-			<!-- php will be used to change link depending on if the user is logged in -->
-			<p class="return_link"><a href="<?php if(!isset($_SESSION['login_user'])) {echo 'login.php';} else {echo 'index.php';} ?>">Return</a></p>
+			<!-- Link that goes back to index page -->
+			<p class="return_link"><a href="index.php">Return</a></p>
```

Comment:
I made the code here smaller since before it was made from a copy of the contact page and that page can be viewed without being logged in while this page cannot